docs/user/04-05-cluster-name.md

```diff
@@ -0,0 +1,51 @@
+# Cluster Name
+
+Kyma Environment Broker (KEB) allows you to set a cluster name during SAP BTP, Kyma runtime provisioning and update operations. 
+To do so, specify the **name** parameter in the provisioning or update request.
+During provisioning, the name parameter is mandatory and must consist of 1–64 characters, using only letters, digits, or hyphens.
+During update operations, the name parameter is optional, but if it is provided, it must follow the same character and length requirements.
+The cluster name is used as the context name when generating kubeconfig files for [Kyma Bindings](05-60-kyma-bindings.md) and the [Kubeconfig Endpoint](03-15-kubeconfig-endpoint.md).
+
+See the example:
```

Comment:
Is this a provisioning request? If it is, please specify it as in the next example.